sw/device/silicon_creator/lib/base/rom_mmio.h

```diff
@@ -0,0 +1,71 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_ROM_MMIO_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BASE_ROM_MMIO_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file
+ * @brief ROM Memory-mapped IO functions, for volatile access.
```

Comment:
Maybe a better name for this is "Absolute MMIO" (and replace the `rom` prefix with `abs`)?

Reply:
If there are no objections I can switch to `abs_mmio_{read,write}{8,32}`

Reply:
Done.